experimental/auth-logic/provenance_wrapper_test.go

```diff
@@ -0,0 +1,53 @@
+// Copyright 2022 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package authlogic contains logic and tests for interfacing with the
+// authorization logic compiler
+package authlogic
+
+import (
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/project-oak/transparent-release/slsa"
+)
+
+func (p provenanceWrapper) Identify() Principal {
```

Comment:
Why is it implemented in the test only. Should not it be in `experimental/auth-logic/provenance_wrapper.go`?

Reply:
yeah I'm also confused by this. If there is something that we are both missing why this should be here, please add a comment.

Reply:
There's an issue for this now so that this can be a kind of resolution: [41](https://github.com/project-oak/transparent-release/issues/41). Re-open this as needed if you need to, though.